pkg/cmd/types.go

```diff
@@ -9,6 +9,19 @@ var (
 	ScanOptions = []string{"openscap"}
 )
 
+type MultiStringVar struct {
+	Values []string
+}
+
+func (sv *MultiStringVar) Set(s string) error {
```

Comment:
perfect, that is what I figured since I didn't see an explicit set anywhere in the other code